code/hud/hudsquadmsg.cpp

```diff
@@ -147,8 +147,9 @@ SCP_vector<player_order> Player_orders = {
 };
 
 const SCP_set<size_t> default_messages{ ATTACK_TARGET_ITEM , DISABLE_TARGET_ITEM , DISARM_TARGET_ITEM , PROTECT_TARGET_ITEM , IGNORE_TARGET_ITEM , FORMATION_ITEM , COVER_ME_ITEM , ENGAGE_ENEMY_ITEM , DEPART_ITEM , DISABLE_SUBSYSTEM_ITEM };
+// note: STAY_NEAR_TARGET_ITEM appears in both enemy and friendly sets
 const SCP_set<size_t> enemy_target_messages{ ATTACK_TARGET_ITEM , DISABLE_TARGET_ITEM , DISARM_TARGET_ITEM , IGNORE_TARGET_ITEM , STAY_NEAR_TARGET_ITEM , CAPTURE_TARGET_ITEM , DISABLE_SUBSYSTEM_ITEM };
```

Comment:
Why are we keeping `STAY_NEAR_TARGET_ITEM`  in `enemy_target_messages`?

Reply:
Because, conceptually, the order should be valid for both friendlies and enemies.  I could see potential uses for either one.  But to your point, yes, the behavior now works for both friendlies and enemies, and I've tested it with both.